view/theme/frio/php/default.php

```diff
@@ -26,40 +26,43 @@
 		<script  type="text/javascript">var baseurl = "<?php echo System::baseUrl(); ?>";</script>
 		<script type="text/javascript">var frio = "<?php echo 'view/theme/frio'; ?>";</script>
 <?php
-	$baseurl = System::baseUrl();
-	$frio = "view/theme/frio";
-	// Because we use minimal for modals the header and the included js stuff should be only loaded
-	// if the page is an standard page (so we don't have it twice for modals)
-	//
-	/// @todo Think about to move js stuff in the footer
-	if (!$minimal && x($page, 'htmlhead')) {
-		echo $page['htmlhead'];
-	}
-	// Add the theme color meta
-	// It makes mobile Chrome UI match Frio's top bar color.
-	$uid = $a->profile_uid;
-	if (is_null($uid)) {
-		$uid = Profile::getThemeUid();
-	}
-	$schema = PConfig::get($uid, 'frio', 'schema');
-	if (($schema) && ($schema != '---')) {
-		if (file_exists('view/theme/frio/schema/' . $schema . '.php')) {
-			$schemefile = 'view/theme/frio/schema/' . $schema . '.php';
-			require_once $schemefile;
+		$basepath = $a->path ? "/" . $a->path . "/" : "/";
```

Comment:
Is there a specific reason this is now double indented?

Reply:
I tried to intend it according to the `html` tags - so it would be better readable

Reply:
I don't like doing this but I understand why you did.